app/models/member.rb

```diff
@@ -1,2 +1,12 @@
+# frozen_string_literal: true
+
+# Member Model
 class Member < ApplicationRecord
+  # SEARCH by SQL LIKE command or Algoria by config
+  def self.search_result(query)
```

Comment:
A few things here:

* Rails specifically provides [`sanitize_sql_like`](http://api.rubyonrails.org/classes/ActiveRecord/Sanitization/ClassMethods.html#method-i-sanitize_sql_like) for cases like this.
* Right now the app is using SQLite, but there has been talk of using Postgres vs. MySQL as the preferred backend. Note that the behavior of `LIKE` in Postgres is case-sensitive, which differs from SQLite and MySQL. With Postgres, you'd likely want [`ILIKE`](https://www.postgresql.org/docs/9.6/static/functions-matching.html#FUNCTIONS-LIKE) here, or possibly to use a [`citext`](https://www.postgresql.org/docs/9.6/static/citext.html) type in place of a standard string type.
* You can use the Hash syntax for the `where` clause to avoid the `query` keyword duplication _(though it's really just a readability balance)_:

```ruby
  def self.search_result(query)
    term = sanitize_sql_like("%#{query}%")

    Member.where(
      "firstname LIKE :term OR lastname LIKE :term OR bio LIKE :term",
      { term: term }
    ).all
  end
```